stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.12.1
+version: 1.12.2
```

Comment:
Bump the minor version here. Another change is going in before yours so this will be `1.14.0`

Reply:
> Thanks for the PR! Could you also add a small blob in the readme about how to enable the stats page using http-snipet.

Does this make sense? For reference, the snipped is
```
http-snippet: |
  server {
    listen 18080;

    location /nginx_status {
      allow 127.0.0.1;
      allow ::1;
      deny all;
      stub_status on;
    }

    location / {
      return 404;
    }
  }```

This means the webserver is only accessible from localhost. We could change the allowed IPs of course, but then we would want it configurable again. IMHO that feature is not worth the increased complexity of the chart. I also dont think that feature is meant to be used outside of local debugging.

Reply:
We discussed on slack on how best to describe the change within the readme. What we settled on was to describe what the stats page is now for those who come from outdated links.